client/my-sites/site-settings/spam-filtering-settings.jsx

```diff
@@ -59,20 +59,17 @@ const SpamFilteringSettings = ( {
 		className,
 		header = null;
 
-	if (
-		! shouldShowOfferResetFlow() &&
-		! inTransition &&
-		! ( hasAkismetFeature || hasAntiSpam ) &&
-		! isValidKey
-	) {
+	if ( ! inTransition && ! ( hasAkismetFeature || hasAntiSpam ) && ! isValidKey ) {
 		return (
 			<UpsellNudge
-				description={ translate( 'Automatically remove spam from comments and contact forms.' ) }
+				title={ translate( 'Automatically clear spam from comments and forms' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Automatically clear spam from comments and forms.' )` *ES Score: 11*